modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -800,8 +920,11 @@ PERF_TEST_P_(BitwiseNotPerfTest, TestPerformance)
     }
 
     // Comparison ////////////////////////////////////////////////////////////
-    EXPECT_EQ(0, cv::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
-    EXPECT_EQ(out_mat_gapi.size(), sz_in);
+    {
+        EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));
+        EXPECT_EQ(0, cv::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
+        EXPECT_EQ(out_mat_gapi.size(), sz);
```

Comment:
Please leave only one `EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));` and remove the rest.